projects/igniteui-angular/src/lib/core/styles/components/query-builder/_query-builder-theme.scss

```diff
@@ -103,48 +117,154 @@
         width: auto;
         min-width: rem(660px);
         background-color: var-get($theme, 'background');
-        border-radius: border-radius(rem(4px));
-        box-shadow: elevation(12);
+        border-radius: $border-radius;
+        box-shadow: var-get($theme, 'elevation');
+        overflow: hidden;
+	    
+	    .igx-chip__ghost {
+		    position: relative;
+	    }
```

Comment:
The chip from the context of the query builder should be targeted in different way.

Reply:
Even if I add a class to the `igx-chip`, I still need to target the inner element containing `igx-chip__ghost` to override its position.
<img width="969" alt="image" src="https://github.com/user-attachments/assets/8064be71-28e8-46a0-a8a3-0d13d0e4cfed" />